src/content/en/updates/2017/04/headless-chrome.md

```diff
@@ -0,0 +1,369 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Getting started with Headless Chrome
+
+{# wf_updated_on: 2017-04-27 #}
+{# wf_published_on: 2017-04-27 #}
+{# wf_tags: chrome59,headless,testing #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: Learn about Headless Chrome, shipping in Chrome 59. #}
+
+# Getting started with Headless Chrome {: .page-title }
+
+{% include "web/_shared/contributors/ericbidelman.html" %}
+
+<style>
+figure {
+  text-align: center;
+}
+</style>
+
+### TL;DR {: #tldr .hide-from-toc}
+
+[Headless Chrome](https://chromium.googlesource.com/chromium/src/+/lkgr/headless/README.md)
+is a way to run the Chromium browser in a headless environment. Essentially, running Chrome without chrome! It brings **all modern web platform features** provided
+by Chromium and the Blink rendering engine to the command line.
+
+Why is that useful?
+
+A headless browser is great tool for automated testing and server environments where you
+don't need a visible UI shell. For example, you may want to run some tests against
+a real web page, create a PDF of it, or just inspect how the browser renders an URL.
+
+Headless mode is available on Mac and Linux in Chrome 59. Windows support
+is coming soon!
+{: .note }
```

Comment:
Remove this and add `Note:` at the beginning. It'll get formatted automagically

Note: Headless mode is available...